app/2fa/server/startup/settings.js

```diff
@@ -2,48 +2,65 @@ import { settings } from '../../../settings';
 
 settings.addGroup('Accounts', function() {
 	this.section('Two Factor Authentication', function() {
+		const enable2FA = {
+			_id: 'Accounts_TwoFactorAuthentication_Enabled',
+			value: true,
+		};
+
 		this.add('Accounts_TwoFactorAuthentication_Enabled', true, {
```

Comment:
Would be good to add a description to this, so users knows that the setting controls everything 🤔

Reply:
You mean like a comment inside the code? Because if you mean the setting I already added a description saying that, I'll send a print:
![new2faprint](https://user-images.githubusercontent.com/49413772/118290691-16375900-b4ad-11eb-8509-03c528c1a0e9.png)

Reply:
Ah, weird. I thought the setting required the `i18ndescription` key to fetch that from the translations 😮